xmipp

```diff
@@ -47,20 +47,22 @@ SCIPION_EM_XMIPP = 'scipion-em-xmipp'
 CUFFTADVISOR = 'cuFFTAdvisor'
 CTPL = 'CTPL'
 GTEST = 'googletest'
+LIBSVM = 'libsvm'
 
 REPOSITORIES = {XMIPP: 'https://github.com/I2PC/xmipp.git',
                 XMIPP_CORE: 'https://github.com/I2PC/xmippCore.git',
                 XMIPP_VIZ: 'https://github.com/I2PC/xmippViz.git',
                 SCIPION_EM_XMIPP: 'https://github.com/I2PC/scipion-em-xmipp.git',
                 CUFFTADVISOR: 'https://github.com/DStrelak/cuFFTAdvisor.git',
                 CTPL: 'https://github.com/vit-vit/CTPL.git',
-                GTEST: 'https://github.com/google/googletest'}
+                GTEST: 'https://github.com/google/googletest',
+                LIBSVM: 'https://github.com/cossorzano/libsvm.git'}
 
 #              {dep-repo:     (branch, check-conf)}
 DEPENDENCIES = {CUFFTADVISOR: ('master', 'CUDA'),
                 CTPL: ('master', None),
-                GTEST: ('release-1.10.0', Config.KEY_BUILD_TESTS)}
-
+                GTEST: ('release-1.10.0', Config.KEY_BUILD_TESTS),
+                LIBSVM: ('master', None)}
```

Comment:
Better assign the release tested (most readable)
` LIBSVM: ('v325', None)}`

Reply:
v325 is a tag, not a branch, and as far as I can see, Xmipp can clone branches.

Reply:
Please try it, from the documentation of git, `-b` should also work for tags:
`--branch can also take tags and detaches the HEAD at that commit in the resulting repository.`

Reply:
I tried and the command that we have to download dependencies does not work if we substitute master by v325

Reply:
Yes, it says me " Repository contains uncommitted changes."